docs/scripts/rmsecl_volumetrics.rst

```diff
@@ -0,0 +1,12 @@
+
+RMSECL_VOLUMETRICS
```

Comment:
I think we need a more concrete example in the documentation

Reply:
Now added.

Reply:
Excellent!